src/engine/configuration.cpp

```diff
@@ -318,6 +318,138 @@ namespace {
 
 namespace openspace {
 
+ghoul::Dictionary Configuration::createDictionary() {
+    ghoul::Dictionary res;
+
+    res.setValue("WindowConfiguration", windowConfiguration);
+    res.setValue("Asset", asset);
+    res.setValue("Profile", profile);
+    res.setValue("PropertyVisibility", static_cast<int>(propertyVisibility));
+
+    ghoul::Dictionary globalCustomizationScriptsDict;
+    for (size_t i = 0; i < globalCustomizationScripts.size(); ++i) {
+        globalCustomizationScriptsDict.setValue(
+            std::to_string(i),
+            globalCustomizationScripts[i]
+        );
+    }
+    res.setValue("GlobalCustomizationScripts", globalCustomizationScriptsDict);
+
+    ghoul::Dictionary fontsDict;
+    for (auto it = fonts.begin(); it != fonts.end(); ++it) {
+        fontsDict.setValue(it->first, it->second);
+    }
+    res.setValue("Fonts", fontsDict);
+
+    ghoul::Dictionary fontSizeDict; {
```

Comment:
No need for the `{` and the extra indentation here

Reply:
I added it to make the code a little more clear since it is dictionaries inside dictionaries, but I can change it if we want. I just thought it made it more readable.

Reply:
I think we could do the same thing with empty lines instead, breaking the different Dictionaries into more visible blocks by having two empty lines before and after for example.
If you really want to keep the additional scopes, putting the `{` on a separate line would be less jarring and look less like a mistake:

```cpp
ghoul::Dictionary fontSizeDict;
{
        fontSizeDict.setValue("FrameInfo", static_cast<double>(fontSize.frameInfo));
        fontSizeDict.setValue("Shutdown", static_cast<double>(fontSize.shutdown));
        fontSizeDict.setValue("Log", static_cast<double>(fontSize.log));
        fontSizeDict.setValue("CameraInfo", static_cast<double>(fontSize.cameraInfo));
        fontSizeDict.setValue("VersionInfo", static_cast<double>(fontSize.versionInfo));
}
```

though I do think that they distract more than they help